Sources/SwiftCompilerSupport/ConsistencyCheck.swift

```diff
@@ -73,7 +77,9 @@ public func _parserConsistencyCheck(
           continue
         }
 
-        print("\(String(cString: filename)): error: \(diag.debugDescription)")
+        diag.debugDescription.withCString { diagText in
+          hook(diag.location(converter: converter).offset, diagText, ctx)
```

Comment:
I don’t think you actually need a `SourceLocationConverter` here because `AbsolutePosition` already contains a `utf8Offset`.

```suggestion
          hook(diag.position.utf8Offset, diagText, ctx)
```